unlock-app/src/components/interface/locks/Manage/elements/Members.tsx

```diff
@@ -11,8 +11,7 @@ import { subgraph } from '~/config/subgraph'
 import { storage } from '~/config/storage'
 import { Placeholder } from '@unlock-protocol/ui'
 import { useWeb3Service } from '~/utils/withWeb3Service'
-
-const itemsPerPage = 30
```

Comment:
Good to store the constant outside of the file but maybe could keep the current `itemsPerPage` pattern with sth like `const itemsPerPage = MEMBERS_PER_PAGE` ?

That will simplify if this is to become dynamic at some point with a selector of how many items per page should be shown